goal_src/pc/pckernel-h.gc

```diff
@@ -345,6 +374,8 @@
 (define *pc-settings* (the-as pc-settings #f))
 (format 0 "PC kernel version: ~D.~D~%" PC_KERNEL_VERSION_MAJOR PC_KERNEL_VERSION_MINOR)
 
+(define *pc-temp-string-1* (new 'global 'string 2048 (the-as string #f)))
+
```

Comment:
Can't use the other `*pc-temp-string*`?

Reply:
No, I use it to construct a string that is passed to a function that then uses `*pc-temp-string*` (i make the filepath that is passed to the read/write function).  I could be wrong but, one of the problems with depending on these global variables to do things :(

Reply:
ok, it's fine then